package.json

```diff
@@ -12,7 +12,8 @@
     "lint:fix": "eslint src --ext ts,tsx --report-unused-disable-directives --max-warnings 0 --fix",
     "preview": "vite preview",
     "predeploy": "npm run build",
-    "test": "npx playwright test"
+    "test:e2e": "playwright test - c playwright.config.ts",
+    "test:ct": "NODE_ENV=development playwright test -c playwright-ct.config.ts"
```

Comment:
What ct means?

Reply:
"CT" need to run component/unit tests - https://playwright.dev/docs/test-components